schemas/mozilla_nimbus_schemas/jetstream/metadata.py

```diff
@@ -42,11 +42,24 @@ class ExternalConfig(BaseModel):
     url: HttpUrl
 
 
+class ConfigVersion(BaseModel):
+    name: Optional[str] = None
+    path: Optional[str] = None
+    revision: Optional[str] = None
+
+
+class ConfigVersionMap(BaseModel):
+    metric_definitions: Optional[list[ConfigVersion]] = None
+    jetstream_image: Optional[ConfigVersion] = None
```

Comment:
Wait where is this used?

Reply:
Whoops! It should be used in the main `Metadata` object in place of `ConfigVersion`, but I got caught up in not liking the name and forgot to paste it down there. 

Thanks for catching! I'll push an update.